src/cpu/x64/jit_uni_fork_dw_conv_kernel_f32.cpp

```diff
@@ -416,30 +664,43 @@ void jit_uni_fork_dw_conv_fwd_kernel_f32<isa>::generate() {
     if (jcp.with_bias)
         mov(reg_bias, ptr[this->param1 + GET_OFF(bias)]);
     mov(reg_kw, ptr[this->param1 + GET_OFF(kw_padding)]);
-    mov(reg_ch_blocks, ptr[this->param1 + GET_OFF(ch_blocks)]);
+    mov(reg_ch_blocks, ptr[this->param1 + GET_OFF(load_work)]);
     mov(reg_ur_w, ptr[this->param1 + GET_OFF(ur_w)]);
 
     Label ch_blocks_tail_label;
     Label exit_label;
 
     int ch_blocks_tail = jcp.nb_ch % jcp.nb_ch_blocking;
+    if (isa > avx2) {
```

Comment:
This is the first time I've seen such a comparison. Not sure how safe it is to make such comparisons in `cpu_isa_t` enum. Better to replace with equality checks.

Reply:
OneDNN developers use such comparison some times, for example: https://github.com/openvinotoolkit/oneDNN/blob/5df68405a5cae691661cb30963d9284d944dbcb7/src/cpu/x64/jit_uni_pool_kernel.cpp#L251
But considering that we only need to check if avx512 is available so that we can use mask registers. I replaced that with checking a specific bit, it will be more reliable I think.